insights/parsers/rabbitmq.py

```diff
@@ -192,3 +206,44 @@ def parse_content(self, content):
                         "auto_delete should be true or false: {0}".format(line))
             else:
                 raise ParseException("Data appears invalid: {0}".format(line))
+
+
+@parser(Specs.rabbitmq_env)
+class RabbitMQEnv(SysconfigOptions):
+    """Parse the content of file ``/etc/rabbitmq/rabbitmq-env.conf`` using
+    the ``Parser`` base class.
```

Comment:
@psachin this would be more accurate if it was ``SysconfigOptions`` instead of ``Parser``.

Reply:
Hi @bfahr Fixed the docstring